scout/parse/omim.py

```diff
@@ -146,13 +146,19 @@ def parse_genemap2(lines):
         parsed_entry["raw"] = line
 
         # This is the approved symbol for the entry
-        hgnc_symbol = parsed_entry.get("Approved Symbol")
+        approved_gene_symbol_col = parsed_entry.get(
+            "Approved Symbol", parsed_entry.get("Approved Gene Symbol")
```

Comment:
We should introduce a set of headings to expect from the input data.

Reply:
I agree!